styles/base.css

```diff
@@ -8,49 +8,39 @@
     --brand-primary-black: #1C1C1C;
     --black: #000000;
     --white: #FFFFFF;
+    --bs-light-rgb: 84, 84, 84;
+    --bs-secondary-rgb: 190, 190, 190;
+    --bs-font-sans-serif: "Roboto, system-ui,-apple-system,"Segoe UI","Helvetica Neue","Noto Sans","Liberation Sans",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji"";
+    --bs-font-monospace: "Source Code Pro Bold",monospace;
 }
 
-* {
-    box-sizing: border-box;
-}
-
-html,
 body {
-    height: 100%;
-    margin: 0;
+    background-color: var(--brand-primary-black);
+    color: var(--white);
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-weight: normal;
 }
 
-:root {
-    --body-padding: 16px;
+
+.container-fluid {
+    --bs-gutter-x: 2rem; /* 32px total gutters, 16px on each side */
 }
 
 @media (min-width: 992px) {
-    :root {
-        --body-padding: 30px;
+    .container-fluid {
+        --bs-gutter-x: 3rem; /* 48px total gutters, 24px on each side */
     }
 }
 
-body {
-    display: flex;
-    flex-direction: column;
-
-    padding: var(--body-padding);
-    background-color: var(--brand-primary-black);
-
-    color: var(--white);
-    font-family: 'Roboto', Arial, Helvetica, sans-serif;
-    font-weight: normal;
-}
-
```

Comment:
Using the `--bs-gutter-x` to set body padding. So this is no longer necessary. This is how the override looks like in the CSS:

<img width="440" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/e3a92ee4-b6da-4865-bfe5-3b41eb0cfeb8">

The default `--bs-gutter-x`(left and right) is 1.5rem, but I'm setting it to 3rem/2rem (48px total/32px total responsively across Desktop/Mobile). 

Here's the 48px gutter on Desktop in effect. There is 24px on each side:
<img width="338" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/1242f38d-5a6e-4f18-b8cd-0fc5e5ab992c">

Reply:
This `bs-gutter-x` variable is used not just here, but in any class concerning container and column widths: 
<img width="863" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/1eea51ae-8cf3-4750-af96-b6189b4f472b">

So it's pretty cool that with CSS Variables, we can now make cascading variable changes across the app without Sass.